service/src/main/kotlin/fi/espoo/evaka/application/ApplicationStateService.kt

```diff
@@ -1486,6 +1495,22 @@ class ApplicationStateService(
             val newStatus = if (sendBySfi) WAITING_CONFIRMATION else WAITING_MAILING
             tx.updateApplicationStatus(application.id, newStatus, user.evakaUserId, clock.now())
 
+            if (skipGuardian) {
+                decisionIds.forEach { decisionId ->
+                    val decision = tx.getDecision(decisionId)!!
+                    if (decision.type == DecisionType.PRESCHOOL) {
+                        acceptDecision(
```

Comment:
Mitäs kaikkee sivuvaikutuksia päätöksen teolla on? Siitä varmaan lähtee sähköposti ja suomi.fi-viesti joten niiden tekstit tarvii tarkistaa. Esim tää ei pidä paikkaansa jos haet pelkkää eskaria eikä oo mitään mihin pitäis vastaa:
<img width="1199" alt="Screenshot 2025-03-28 at 12 34 49" src="https://github.com/user-attachments/assets/4876a790-0fbd-41fe-8fb9-795391e0cb47" />

Reply:
Toi EvakaEmailMessageProvider on mun nähdäkseni myös default toteutus jota saattaa käyttää myös kunta X, niin en oo varma onks se hyvä et laittamalla feature flagin päälle softa toimii oletuksena väärin. Lisäks tuohon interfaceenkin tarvitaan muutoksia koska jos lähetetään pelkkä valmiiksi hyväksytty eskaripäätös niin sille halutaan eri teksti eikä pelkkä language-parametri mahdollista iffittelyä.